### ISSUE TITLE
Training/Testing datafiles for finetuning

### ISSUE THREAD

**Issue by sefinch** [INK-USC/Reflect#1]
Finetuning blender expects `all_train_responses.json`  and `all_test_responses.json` but the only data that is provided is `organized_Reflect_9k_responses.json`. Based on the agents.py code, the format of `organized_Reflect_9k_responses.json` is different than that of the expected files for finetuning. Will training/testing data files themselves be released, or at least the preprocessing code to convert `organized_Reflect_9k_responses.json`?

**Comment by yana-xuyan** [comment]
I don't think the authors collected training data in this dataset. It's just a test set from what I understand...
